<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sur</class>
 <widget class="QDialog" name="sur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>                          楼层图搜索</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>楼号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="floorComboBox">
       <item>
        <property name="text">
         <string>A1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B3</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>层号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="roomComboBox">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
